pandas/io/excel/_odswriter.py

```diff
@@ -110,6 +113,14 @@ def _write_cells(
         )
         from odf.text import P
 
+        if notes is not None:
+            raise NotImplementedError(
+                """
+                Notes are not supported by the odswriter engine,
+                see https://github.com/mmulqueen/odswriter
```

Comment:
I don't understand the reference to the linked repo. pandas uses https://github.com/eea/odfpy for writing ODS files.

Reply:
Hey, I might be wrong but I believe we went to pandas documentation when searching for all the engines pandas currently uses and found this for ExcelWriters, in here explicitly links https://github.com/mmulqueen/odswriter as the engine to write ods files. 

https://pandas.pydata.org/docs/reference/api/pandas.ExcelWriter.html,

Reply:
yes, `odfpy` is indeed used but for reading Open-Document type files (https://pandas.pydata.org/docs/reference/api/pandas.read_excel.html#pandas-read-excel). The writing operation is taken care by that `odswriter` as my colleague referred.

Reply:
Thanks @diogomsmiranda @Dacops. I can see why now. The docstring in `pd.ExcelWriter` is incorrect (looks like was added in #47568).

1. If you check the `engine_kwargs` section, it mentions `odf.opendocument.OpenDocumentSpreadsheet(**engine_kwargs)`. None of this construct is available in the linked `odswriter` library, but is available in the `odfpy` library instead https://github.com/eea/odfpy/blob/574f0fafad73a15a5b11b115d94821623274b4b0/odf/opendocument.py#L842.

2. If you check what extra libraries pandas installs when you run `pip install pandas[excel]`, you don't see odswriter listed there. https://github.com/pandas-dev/pandas/blob/dd87dd3ef661ac06e30adc55d25a1f03deda3abf/pyproject.toml#L69

cc @rhshadrach

Reply:
Oh I see, I'm glad it could be useful to find the mistake. We'll change the link to the correct one, or even better check if `odfpy` has compatibility with notes, and adjust accordingly.

Reply:
After looking through the codebase for mentions of comments or notes in cells I didn't find anything, although I did find a couple discussion about the current state of the project, mentioned as "dead".

In the mean time I've fixed the exception to link the right library.

Reply:
> current state of the project, mentioned as "dead".

I think that's fair. The last commit on that repo seems to be 3 years ago.